src/uwtools/tests/config/test_validator.py

```diff
@@ -13,7 +13,7 @@
 from uwtools.config import validator
 from uwtools.config.formats.yaml import YAMLConfig
 from uwtools.logging import log
-from uwtools.utils.file import resource_pathobj
+from uwtools.utils.file import resource_path
```

Comment:
Since we've now eliminated (I think) `str` paths from the codebase, paths are always `Path` objects, so `pathobj` is a somewhat redundant name. Simplified.